### PR TITLE
feat: added marketing banner

### PR DESCRIPTION
**Pull request by okhaimie-dev**
## Changes

- Added Banner component for newsletter signup

CleanShot 2023-09-20 at 08 02 25@2x

### CODE REVIEW

**Review by wolovim — APPROVED** [review]
looks nice to me! non-blocking ideas: 
1. if not too much trouble, would be nice to be able to dismiss the banner.
2. we could tease v2 in the text, e.g., "Sign up for the newsletter for updates on the redesign!" or something to that effect.

take it or leave it 🤝

**Comment by kempsterrrr** [comment]
Looks good @okhaimie-dev - nice work.

Agree with @wolovim on both points, The suggested language to replace the message from me is...

"Academy V2 coming soon 👀... Subscribe to our newsletter for updates!"

Good to build some fomo/excitement.

File: components/Banner.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    <Center
+      py="2"
+      px="3"
+      bgGradient="linear(to-r,cyan.700, purple.500)"
+      color="white"
+      textAlign="center"
+    >
+      <Flex align="center" fontSize="sm">
+        <Text fontWeight="medium" maxW={{ base: '32ch', md: 'unset' }}>
+          Join our newsletter for exclusive updates!
```
Comment (kempsterrrr) [inline components/Banner.tsx:14]: Update message to something like...

Acamdey V2 is coming soon 👀... Subscribe to our newsletter for updates!

Comment (okhaimie-dev) [reply components/Banner.tsx]: @kempsterrrr @wolovim 

Implemented y'alls suggestion.

**Comment by kempsterrrr** [comment]
Looks good to me!

**Review by kempsterrrr — APPROVED** [review]
Update looks good to me mate

**Pull request merged**